### DISCUSSION TITLE
Niko hue button support question

### DISCUSSION THREAD

**Discussion by jeroenvdd** [Koenkk/zigbee2mqtt#10809] [Devices]
hi guys,
simple question, I've got a new button
https://www.zigbee2mqtt.io/devices/PTM_215Z.html  (the Niko 91004)

that was under supported devices:
![image](https://user-images.githubusercontent.com/6975276/149620495-aca51fdb-102e-4bd5-9d71-dafba6cd296d.png)

But is this supported when you use a slaesh stick?
'cause it has a wierd description about needing a hue device.
to be clear: I don't have any hue stuff in my house

so a) do i need to get a hue product to make this work? (maybe just 1 light or something in range? or do i need the bridge?)

or is their a way to get this working? (if not maybe we should put a big exclamation mark next to the supported page?)

**Comment by Koenkk** [comment]
you need 1 green power router in range (which hue bulbs are, but also switches)

**Reply by jeroenvdd** [reply]
alright, so if i buy a hue GU10 spot link that to my z2m i should be able to link the switch, if they are relatively close?
(without needing a hue bridge)

and maybe ad another exclamation mark or something on the supported product overview page?

PS. thx for making an awesome product Koen, you don't hear this enough :)

**Reply by Koenkk** [reply]
yes than it works.

**Reply by jeroenvdd** [reply]
> yes than it works.

On the openbaar forum someone got it working without any other hue device? With a conbee stick any ideas what could make it function within a conbee that a slash stick does not have

**Reply by Koenkk** [reply]
Sleash uses a different software stack (zstack from Texas Instruments) which does not support his feature yet.

**Reply by sjorge** [reply]
There are other devices that act as a GP proxy. Some of my devices that are !Hue that have the clusters:
- Ubisys S1
- Ubisys S2
- Ubisys J1
- Ikea GU10
- Ikea FLOALT
- Ikea GUNNARP

So check the cluster tab of some of your existing routers, you might not need the Hue bulb(s)

**Reply by jeroenvdd** [reply]
> There are other devices that act as a GP proxy. Some of my devices that are !Hue that have the clusters:
> 
> * Ubisys S1
> * Ubisys S2
> * Ubisys J1
> * Ikea GU10
> * Ikea FLOALT
> * Ikea GUNNARP
> 
> So check the cluster tab of some of your existing routers, you might not need the Hue bulb(s)

the guy on homeassistant fora has a conbee and 3 aqara motion sensor and then this switch that's the strangepart, no other zigbee stuff..
![image](https://user-images.githubusercontent.com/6975276/149660990-577c9c1a-063c-4cf0-a64d-573b4a4aad88.png)

**Comment by jeroenvdd** [comment]
I got this working by adding 1 hue spotlight to my house,
thx for the info

(although I'm still convinced the warning on the support overview page should be added)
